Interface/InterfaceExample/objects/func.go

```diff
@@ -0,0 +1,47 @@
+package objects
```

Comment:
Please have the methods of the struct in respective files, 
it is creating a dependency on single file.